site/content/en/blog/releases/1.37.0.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "1.37.0 - Counters And Lists Alpha Release and Multiple Feature Promotions!"
+linktitle: "1.37.0"
+date: "2023-12-19"
+---
+
+This is the 1.37.0 release of Agones.
+
+This release sees the first milestone of a over-a-year series of work on the functionality of arbitrary Counters and Lists attached to each GameServer!
+
+We’ve also upgraded the following features: CountsAndLists to alpha, FleetAllocationOverflow to Beta, and PodHostName, ResetMetricsOnDelete, and SplitControllerAndExtensions to Stable - with defaults below.
+
+- CountsAndLists: This feature flag provides functionality for a configurable set of Counters or Lists be attached to GameServers with attached allocation and autoscaling functionality to provide a flexible implementation that can handle a wide range of use cases such as player tracking, high density game server room tracking, and could even be used for game specific game server allocation prioritization.
+
+- Documentation is a little sparse in this release, but will grow! We’re focusing on the reference material first and will then expand into more in-depth guides and integration patterns.
```

Comment:
This and the next paragraph shouldn't have new bullets in the bulleted list. Let me see if I can tinker with the markdown to make it look a bit better.

Reply:
Does it look good?

https://screenshot.googleplex.com/5hYyMBcWTg7ryJq

Reply:
Yes - that's what I was going for :)

Reply:
Quick question:

Do we need to delete the `data-proofer-ignore` attribute from the files 
`site/content/en/docs/Guides/Client SDKs/rust.md` and `site/content/en/docs/Third Party Content/examples.md`?

https://screenshot.googleplex.com/9Wx4Pt9mCo6AmtU

https://screenshot.googleplex.com/97QN9ZLDoyQgLav